html/418-request-denied.shtml

```diff
@@ -0,0 +1,15 @@
+<!--#if expr="${VERBOSE_ERROR_PAGES}=true" -->
```

Comment:
Should the var be lowercase?

Reply:
According to what I have read the property name should be uppercase

Reply:
Okay.
It looks like it only matches is the environment is literally the string "true". i.e. It is case-sensitive.
It would be nice if it could be more lenient somehow but as long as it is documented it should be fine.